### PR TITLE
[DOCFIX] Add the Chinese version of Configuring-Alluxio-with-GCS.md

### PR DESCRIPTION
**Pull request by Popeye13**

### CODE REVIEW

**Comment by ifcharming** [comment]
ok to test

**Comment by ifcharming** [comment]
LGTM, thank you for the awesome translation!

File: docs/cn/Configuring-Alluxio-with-GCS.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+{% include Common-Commands/copy-alluxio-env.md %}
+
+而且，为了在Alluxio上使用GCS, 需要创建一个bucket（或者使用一个已有的bucket)。你应该注意你在这个bucket里准备使用的目录，你可以自己在这个bucket里面创建一个新目录，或者使用一个已有的目录。在这个指南中，我们将GCS bucket取名为`GCS_BUCKET`，bucket中的目录取名为`GCS_DIRECTORY`。
+
+如果你刚接触GCS，请先阅读GCS[文档](https://cloud.google.com/storage/docs/overview)。
+
+# 配置Alluxio
+
+为了配置Alluxio以使用GCS作为其底层文件系统，需要修改`conf/alluxio-env.sh`文件。第一个需要修改的地方就是指定一个已有的bucket以及bucket中目录作为底层文件系统。你可以通过向`conf/alluxio-env.sh`文件添加以下代码来指定：
```
Comment (uronce-cc) [inline docs/cn/Configuring-Alluxio-with-GCS.md]: ...中的目录...

**Comment by uronce-cc** [comment]
@Popeye13 Thanks for translation! Left one minor comment.

File: docs/cn/Configuring-Alluxio-with-GCS.md (hunk 1 above)
Comment (Popeye13) [inline docs/cn/Configuring-Alluxio-with-GCS.md]: Updated

**Comment by Popeye13** [comment]
@uronce-cc Thanks for your review, updated.

**Comment by yupeng9** [comment]
LGTM. Thanks!

**Pull request merged**